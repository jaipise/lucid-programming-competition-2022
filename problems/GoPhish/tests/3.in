100
ndruZuyBMLUWFcDexuRbB eMHSMm
DfFiOQHHUJRBQaCGMkOo
BhjCChtZpeFxDnOVqRNErVjEKJxHOjQsC
zbsARJId
fYLVEyjbGmAvCpt
eGYJKTzkknZVubUKhAFlpWi
RxhRxmplXpKEwVZNmEWqdnVZWZVLvxbGvK
zJouoryJkrArzntpepKoUTVWmaLHHNtKCzsnUfyOdzBFroNgbgk
KZxJGFLIMYgcLTp
StnvE
N UFpAX
CWzIFaxbpSNvgAPGogxzwmPWiejBGLOLRQCLiFLRrrDX
QSFszKLFhvVEyWbbwOxXBVEg
KzskLBUywkwCvJZiKVfAIBXynuzPGiDs
NdezhxRAaNPUoxIgjgqdUgcARyJJbSVUEdZEpqi
VKeJDSOJZJEtMvLcDGVCkGsNqoEKgxDAHGLpxYCoMZksPytF
iTQckVyApYRtzmJ
mEEIccXCOvBKna DREpuBysxiWKxXhYNOUnHMBFFmkrVndPdhdGDsnk
CESuuBDCCzcPfwOQqWwpwONXIqZhhLyLsGSBzEfDLaLIBeOQW
MYKiJGYMUybJNYwBCLOZWvKSfHbIu
bEZppQbDwsyPINmyRibkyMmjOnAzAnqtjeLl
PFJQUUiYBki PDyQUf
hndfXVksvUasburZMeKKHKRFZBSgvXKAQkwoLMVTZyadEhptHH
gdhiNlbLmJnIn
UWOIJkKIrOIVySy UMUwVObHcJoLNNzkGgXGsvuzYyC
AltlIBTCpgNSiqnqOsMULGuSpwGqsCeZMNYIwUgjd
mHYtEPBCGc VjVfYnPiEslDIzlNCxyThkWQsQXIqLtv
UYZmFusohvuXsQhlhwRP
C FjcwJHrUOr
xvtOydCTWgdOzcYoNJyptsJxlwNiUwrhaJgPZ
VRTXTVcSGdQjSKKy
FsqLgXub
FZfaAIIM
yBKQhkbwITZixBzepAxlYDpExYcKTCRPPOnpeojpZWPwM
hBPpONJCwzJsxfRlKhM bARojbJrYuoChxvgKDTmsgEQNRWGSWq
giyzwx Z
zlSbnEzaEwMKPYthFGgOnecEtjnYAHOwHpEkotbRSVKFE
DqZubUZtLuIvNGwathFzcngClDhikdkVCx
OXGw eXQcSQCJmBYJF
VfGXbXDVFWciRgXTgFjtLImmqfLOvFLUHTBQBjbTXGQYcymd
UtgXAkVgHfkyoLFIrCFoUVvfePZNXJGFS
VkMeBOOtoggwyNiGbqAbcYlyBwHAgwpb
IxGWXzw
ciJZHblVBnJtzurlKYdgpGNflhDhKoJcPlrq
kfejqJsPijPjvz
aOqQEGKdg KbyEFZciRjUHYrQ
IvpjYxLskDzmxhqHjHaCPCUnm
DAnoqmXXrvms klzON
WKTCFEValttyN cZSNESpm
igzCCwTBqlLliJgYUzQFBjdqCwkOinSqIniBhhLKV
IqXHyGSiViwyPgSEwsztdziT
WpiwIaCwetijTTtKeCHwQabtQmKrLzKmoyWDbzVAepSQlWgxTzxehh
dgPSyIfaGQkNim YrCB
canxVlWrGV
EvwcCva
svAPbGSqg
bYAoiBRlbhLDWscQzfFoBILdpGzXqHMEWGpiNTUKvwPlZvEWlHhMkq
zgwnBSMTTFWKzWLMrXfiGbuPpzcOwWiafcrzolxkOEleu
EESJXMAhUbBBppyqcKigeXMLKilrcCczYKcnaYyRFwzWpTguLmoJvF
Vh GXGlWZM
qzitGZGPWCnlTR
DBikIQzNlwGhGxWxGRxEvcQlAxWs
ssNJsXmBSvrVOuZdwEznzrVnOxoDCFG pZG
bbNHgRnmrECffpudiqRHLjZXENgcFrPDOOOmqMtTkaGTtSKZlV
Il oJPGprTVbjMuXJmqOMGDwyTbHraSGrnXJLmhxoiQgCjednxqzH
wwReRWVlGQrIypbQechbGHbknIHiIGGTknvujapD
RXWJZTSkwowNpNc
Ku Azl
RWCzZWyYoUbtDOWGjiEvCJErXMfViOgByQ
OyMtDHVFWddnkzQYZQGIPilxyBRHCBgzjXAPJWaV
hfwfnsAobaIVxokeWN
yoRtLqKcVfriDFzgGBfkbdfJIuHLHUHOdoQFig
gzQeHNJtdwijMtCuAhNmLnFVnDDutaG
CkTsWpecXKplGLa
zWCpbE butcwMfrzcvOxPKclPFsTErxCKLpEBtGqSIC
gxsaELZhHsSTFyUEJYyxGjihtFjleMpfvhuMvRYIZ
TfUUNDDwJrhGBPPD
uNPF WwEYoMAvCF
aHjjBqvYWtF
gKQNwPltULwjTAcbPBmjkQHmoJoWIdXODVpeSlMKsyz
yBcqRCeVWqIfFUHiTPzHndRxHBRSlDSlRynLeYmtzfGZQS
gZTKuoGidxNYoITtzY kmVmSQRjJOhnOWyVrQlcnE
NLmvtYUEBoarkakswftxdEiRqUJWXOoTxImUcINmwcMLJBef
XIcNgPphVQptnhyHdIdTthzaJQZTBMVzZNFPWLLSN
retqhfzcCFpneul
PtxQQJcBWNFQMyBYMiqDIGvRfKZfSyB
esQimvIIQfAIDapafIQDFeNKbaIQGJTmDgFCDEQvDgiJO
jxlanigytVulCgEBqv
NRreVOnlzsdhiyMUxienaVXhSgmSVylvJJJvPPQzxC
PCsVwoNPgvOTtUerGSYGR hZvjOguqeBMU
DpVyfrBZUZJQJmUzGesrQucsYygcPXQlL
hEkpAYYLnWdgxaCHXSidCvMQgHMZSfapvkvAtiHkpnsparVt
UuVdgrSbdIwIAWyuYoaeXRZgrxQUx
CohZ gxTix
NYeCeAmPLIplvbEhKZxglkBYMxQMVNBvdB
ADfdNQYoNSXuAdimyDvrUSadMXxabriwbSac
e ckvqFcEqkbdUXywOGiZWHWlOJvDKIXXngPbBGYIuLQURUkAXNss
oZsYfbuW WMzmrGRhjonJupmnwh
HiYopIVLwaTALXnPZoExPzBDTuUWwTT
qovQoLdrWvHrsSubg
100
qzitGZGPWCnlTR
NDRUzUYbmluwfCmdEXUrBb EmhsmM
CFRvVQwluHTvlhSOWIlbphoekJgUHYnlPmrPojWh
gKQNwPltULwjTAcbPBmjkQHmoJoWIdXODVpeSlMKsyz
nyEcEaMpliPLVBeHkzXGLKbymXqmvnbVDb
NLmvtYUEBoarkakswftxdEiRqUJWXOoTxImUcINmwcMLJBef
QARSBfOgxIvUmcmbUlXiqguDKD
v gxgLwzm
SWXnjRAqkhUddoqJtcmDCLuOwAniTFy
wktcfevALTTYn CzsnesPM
hBPpONJXwzJsxfRlKhM bARojbJrYuoChxvgKDTmsgEQNRWGSWq
FIBccgCsNsFjJCQjiAvwQxPBinXRIsIxGJW
UUCuOFeuZpcFyrbgEDJ
H gxgLwzm
KMbomevKY XCy
xctxkwvaGaeMZYgkp
kU aZL
yoRtLqKcVfriDFzgGBfkbdfJIuHLHUHOdoQFig
VNh GXGlWZM
ojFzcEFBlnVxnGUY
DjdpJCtkfKtn bZNRtkpXwPRYhGYtxMaVHDhFnVLg
yBKQhkbwITZixkzepAxlYDpExYcKTCRPPOnpeojpZWPwM
ZLsBNeZAcWmkpyTHfgGoNECeTJNyahoWhPeKOTBrsvkfe
GZqEhnjTDWIJmcUaHnMlNfvNddUTAg
HPwCPyYeYlVWRZqcXbcUMIcdi
aHjjBqvYWtF
ATAmEfkTFPuyRZgWuFxwUFMHevOpEWdFEcZtLsBMlbUVLZvqmrGewu
uwoijKkiRoivYsY umuWvoBhCjOlnnZKgGxgSVUZyYc
HiYopIVLwaTALXnPZoExPzBDTuUWwTT
CESuuBDCCzcPfwOQqWwpwONXIqZhhLyLsGSBzEfXLaLIBeOQW
C FjcwJbHrUOr
iWYmyrMe jpTPvbTlxQS
pUSvWOnpGVotTuERgsygr HzVJoGUQEbmu
MYKiJGYMUybJNQwBCLOZWvKSfHbIu
ccsicXgBKSWmZmziHQTzMLnyNADhurrHrQMHFjSnOtAzfDsKPGsUr
iXgwxZW
vzfkBulQciWRdmxmmMvwnFeNuqJSDtYro
IqXHyGSiViwyPgSEwsztdziT
RXWJZTSkwowNpNc
zbsARJId
HiYopIVLwaTALXnPZoExPBDTuUWwTT
VM InqndFyGnbnM
VC qajEVhDEVodqdeBjzlEBiaInSPaTVzVLhEfAJRBby
AhJJbQVyLTf
uNPF WwEYoUAvCF
LXkdRo mPNJAYoIqCpTMKedEotnbZphJbUS
qsfSZklfHVveYwBBWoXxbveG
PI BlxT
FylveYJBgMaVcPT
gKQNwPltULwjTAcbPBmjkQHmoJoWIdXODVpeSlMKsyz
yyQyXaQPNWsnWekrvSlQaVjAichWdoDqYOcXYcfQcKaXquqG
DpVyfrBZUZJQJmUzGesrQucsYygcPXQlL
NYNFIB rLf
wktcfevALTGYn CzsnesPM
xFdKjAdaInlYStQhja
NDRUzUYbmluwfCdEXUrBb EmhsmM
n ufPax
GnoFDJervqfrXdh bJMTBLozqW
pp sNpXJpEGZhpAASHiuNykEKhn
uNPF WwEYoMAvCF
kzXjgflimyGClt
wPIWiAcWETIJttTkEchWqABqMkRlZkMOYwdBZvaEPsqLwGXtZXEHH
QKDKrSxpZmvyYkNayOWCDTAUjjdUgsjuoIDzFbjpDOAWfGqYoBKNf
xanxVlWrGV
NlcgSMkcizLWYecvCEzxuJwCJIlanwJnbCTKHvak
DNyJEqBbQApJMWvQuVIRLGpzFXtlXTcCqzB J
TzZuXGhhGeTqNtRxgovSOEqtKQrznVvxZGkANORzGofccWzxC
uTGxaKvGhFKYOlfiRcfOuvVFEpznxjgfs
dySGAhJfk CxgkdjY
CANRvLwRgv
LeUEMDVnxFfgXxbuBiXWXoKVQLwCaYBWiEdRIYJrQfydsHDKkrP
mpmccgTDYxxDfHWjv
KFEJQjSpIJpJVl
yBcqRCeVWqIfFUHiTPzHndRxHBRSlDSlRynLeYmtzfGZQS
sxqPBcmtRpWXuSEqBAsPXXcQNRzzduwRcaSdyMPeP GUv
xZbFJwkCPPhuHWHYQPdennU Ny
TfUUNDDwJrhGBPPD
EvwCva
UWOIJkKIrOIVySy UMUwVObLcJoLNNzkGgXGsvuzYyC
VKeJDnOJZJEtMvLcDGVCkGsNqoEKgxDAHGLpxYCoMZksPytF
CESuuBDCCzcPfwOQqWwpwONXIqZhhLyLsGSBzEfDLaLIBeOQW
UUCXBJuaxKqnUvJkuKwMBHHFEWOKdCpttAvWNi
tEqcAM TqHlvCktWjnhkbviBltTiI
uIvqqq ekftPCkRGqAhHvRtiVHpmrUutl
iL OjpgPRtvBJmUxjMQomgdWYtBhRAsgRNxjlMHXOIqGcJEDNXQZh
oxgW ExqCsqcjMbyjf
gxsaELZhHsSTFyUEJYyxGjihtFjleMpfvhuMvRYIZ
svAPGSqg
rXHrXMPLxPkeWvzniMewQDNvzwzvlVXBgVk
gagMAjEmmcAdZuij
ZrTOCyrChEzembyCEwFgEnHJnsnglZicEooWimaOxIVcPzKfxhEWCh
fSQlGxUB
yQy JsHhD
IvpjYxLpkDzmxhqHjHaCPCUnm
glhiNlbLmJnIn
EVrCgBrQHMfB iFjaOslUyWrvJ
XIcNgPphVQptnhyHdIdTthzaJQZTBMVuZNFPWLLSN
nDEZHXraAnpuOXiGJGQDuGCarYjjBsvueDzePQI
nyEcEaMpliPLVBeHkzXGLKbymXqmvnbVDb
bXuiZcrGkrrYkvFijFNZiluJYWVYXUsoaLyzuzMyxbyvQLGyxAJ
