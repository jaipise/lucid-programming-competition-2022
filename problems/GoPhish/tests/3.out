EMPLOYEE
BOT
CUSTOMER
EMPLOYEE
EMPLOYEE
EMPLOYEE
CUSTOMER
BOT
CUSTOMER
EMPLOYEE
BOT
CUSTOMER
CUSTOMER
BOT
CUSTOMER
CUSTOMER
EMPLOYEE
EMPLOYEE
BOT
CUSTOMER
CUSTOMER
BOT
BOT
BOT
CUSTOMER
EMPLOYEE
CUSTOMER
EMPLOYEE
EMPLOYEE
BOT
BOT
CUSTOMER
BOT
BOT
CUSTOMER
EMPLOYEE
CUSTOMER
EMPLOYEE
EMPLOYEE
EMPLOYEE
BOT
CUSTOMER
CUSTOMER
BOT
BOT
CUSTOMER
EMPLOYEE
CUSTOMER
EMPLOYEE
EMPLOYEE
CUSTOMER
EMPLOYEE
CUSTOMER
BOT
CUSTOMER
EMPLOYEE
EMPLOYEE
CUSTOMER
CUSTOMER
EMPLOYEE
BOT
BOT
CUSTOMER
BOT
CUSTOMER
CUSTOMER
CUSTOMER
EMPLOYEE
CUSTOMER
BOT
CUSTOMER
CUSTOMER
BOT
EMPLOYEE
CUSTOMER
CUSTOMER
EMPLOYEE
BOT
BOT
BOT
EMPLOYEE
CUSTOMER
CUSTOMER
CUSTOMER
EMPLOYEE
EMPLOYEE
EMPLOYEE
BOT
BOT
CUSTOMER
CUSTOMER
EMPLOYEE
CUSTOMER
BOT
BOT
CUSTOMER
BOT
EMPLOYEE
EMPLOYEE
CUSTOMER
